tap_tiktok_ads/client.py

```diff
@@ -67,7 +69,19 @@ def check_access_token(self):
 
         if error_code != 0: # `0` error code indicates successful request
             raise TikTokAdsClientError(message, response) # raise the exception with the message retrieved
-        return bool(resp.get('message') == 'OK')
+        # if the check_access_token() succeeds, then check the account access with the account ids provided in config.
+        if resp['message'] == 'OK':
+            self.__verified = True
+            headers = {
+                "Access-Token": self.__access_token
+            }
+            params = {
+                "advertiser_ids": self.__advertiser_id
+            }
+            # Call the advertisers API with the account ids to check whether the accounts ar valid or not.
+            adv_response = self.get(path='advertiser/info/', headers=headers,
```

Comment:
We need to do separate calls for each account_id else How do we get to know the culprit?

Reply:
The API itself returns which advertiser_ids are invalid
So we won't need individual calls